### DISCUSSION TITLE
Integrating Huawei EMMA in EVCC

### DISCUSSION THREAD

**Discussion by Makki87** [wlcrs/huawei_solar#781] [Setting up the integration]
Hey all,

from another forums I learned that Huawei EMMA provides modbus registers that can be used for the integration into EVCC. Are there any practical experiences with EMMA already?

Many thanks & br

Markus

**Comment by tmenguy** [comment]
hi, 

could you post here the forum ? or how they managed to connect to EMMA?

Thx!

**Comment by Makki87** [comment]
Hi there,

The link to the forum:

https://www.photovoltaikforum.com/thread/229493-huawei-emma-a02/?postID=3802354#post3802354

Haven‘t read anything, but I am also still wondering if EMMA works with EVCC or not.

**Reply by HFB2022** [reply]
Hi, yes its working for me. I had to change the evcc.yaml according to: https://docs.evcc.io/docs/devices/meters#emma 
Thats it.

**Comment by tmenguy** [comment]
Thx
